server/services/BarChartTableService/index.js

```diff
@@ -46,9 +41,19 @@ class BarChartTableService {
   }
 
   _graphTableColumns = () => [
+    {
+      dataProperty: site,
```

Comment:
if this is the only place this is used maybe we just hard-code it?

Reply:
ya hard coding it for sure,

Reply:
[5b4b2a4](https://github.com/AMP-SCZ/dpdash/pull/695/commits/5b4b2a4823862cecfb04b595c18aff35b4b2c9b0)